Software-test-design-guide.md

```
@@ -0,0 +1,469 @@
+# A Practitioner's Guide to Software Test Design 実践ガイド
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)